omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb

```diff
@@ -73,6 +73,12 @@ http {
   }
   <%- end -%>
 
+  <% if node['private_chef']['data_collector']['root_url'] && node['private_chef']['data_collector']['token'] -%>
+  upstream data-collector {
+    server <%= URI.parse(node['private_chef']['data_collector']['root_url']).host %>:<%= URI.parse(node['private_chef']['data_collector']['root_url']).port %>;
```

Comment:
This would mean multiple Automate (Visibility) instances to be proxied by one chef-server? I doubt it.... 🤔 

We'd have to have some more logic in there then -- to which one should the request be routed...?